docs/detections/detection-engine-intro.asciidoc

```diff
@@ -68,16 +68,26 @@ To make sure you can access Detections and manage rules, see
 Cold tier is a {ref}/data-tiers.html[data tier] that holds time series data that is accessed only occasionally. In {stack} version >=7.11.0, {es-sec} supports cold tier data for the following {es} indices:
 
 * Index patterns specified in `securitySolution:defaultIndex`
-* Index patterns specified in the definitions of detection rules
-* Index patterns specified in the data sources selector on various {security-app} pages
+* Index patterns specified in the definitions of detection rules, except for indicator match rules
+* Index patterns specified in the data sources selector on various {es-sec-app} pages
 
 {es-sec} does NOT support cold tier data for the following {es} indices:
```

Comment:
No major issues in this section. Only flagging because I've seen "NOT" bolded for emphasis in other docs.